amps/ags/rm-community/rm-community-repo/unit-test/java/org/alfresco/module/org_alfresco_module_rm/hold/HoldServiceImplUnitTest.java

```diff
@@ -306,6 +307,88 @@ public void setHoldReason()
         verify(mockedNodeService).setProperty(hold, PROP_HOLD_REASON, HOLD_REASON);
     }
 
+    @Test
+    public void setHoldDeletionReasonForNodeDoesNotExist()
+    {
+        // setup node service interactions
+        when(mockedNodeService.exists(hold))
+            .thenReturn(false)
+            .thenReturn(true)
+            .thenReturn(true);
```

Comment:
You don't need the two `true` statements now.

Reply:
There are similar extra `thenReturn` statements below.

Reply:
Removed extra statements.